src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+const SearchContainer = styled.div`
+  position: absolute;
+  top: 15px;
+  width: calc(100% - 30px);
+  left: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 10;
+`
+
+const SearchBox = styled.div`
+  width: 100%;
+  height: 45px;
+  display: flex;
+  align-items: center;
+  background: white;
+  padding: 10px;
+  border: 1px solid #FCEF5C;
+  border-radius: 10px;
+`
+
+const SearchIcon = styled.i.attrs({
+  className: 'fa fa-search',
+})`
+  color: gray;
+  font-size: 20px;
+`
+
+const SearchText = styled.input`
+  border: none;
+  font-size: 20px;
+  outline: none;
+  padding-left: 10px;
+  padding-right: 10px;
+  max-width: calc(100% - 20px);
```

Comment:
```suggestion
  min-width: calc(100% - 20px);
```

Use `min-width` to make this use the whole available with on desktop